modules/mw.cx.TranslationMdwiki.js

```
@@ -199,124 +227,202 @@ async function get_from_medwiki_or_mdwiki_api(title, tr_type) {
+	let html = data.html || "";
+	// ---
+	if (!html || html === "") {
+		console.log("get_mdtexts_2024: not found");
+		return false;
+	}
 
-	var html2 = data.html.replaceAll("&#34;", '"');
+	var html2 = html.replaceAll("&#34;", '"');
 	const matches = html2.match(/Mdwiki_revid"\},"params":\{"1":\{"wt":"(\d+)"\}\}/);
 
 	if (matches && matches[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)